pyclesperanto_prototype/_tier8/_deskew_y.py

```diff
@@ -41,18 +41,20 @@ def deskew_y(input_image: Image,
 
     from ._AffineTransform3D import AffineTransform3D
     from ._affine_transform import affine_transform
-
+    import math
+    
     # shear in the X plane towards Y
     transform = AffineTransform3D()
     transform.shear_in_x_plane(angle_y_in_degrees=angle_in_degrees)
```

Comment:
```suggestion
    transform.shear_in_x_plane(angle_y_in_degrees=90 - angle_in_degrees)
```
Does that make sense here?

Reply:
Haha, it make so much sense to add it here. 
Both the shear and the rotate have consistent angles as well, as in they both use` 90 - angle`